src/dq_suite/df_checker.py

```diff
@@ -1,35 +1,39 @@
 import json
 from jsonschema import validate as validate_json
-
 from databricks.sdk.runtime import *
-
-from pyspark.sql import DataFrame
-
+# from rule_val import handle_dq_error, handle_unexpected_error
 import great_expectations as gx
 from great_expectations.checkpoint import Checkpoint
-
-from dq_suite.input_validator import validate_dqrules
-from dq_suite.output_transformations import extract_dq_validatie_data
-from dq_suite.output_transformations import extract_dq_afwijking_data
+from pyspark.sql import DataFrame
+from rule_val import handle_errors
```

Comment:
I see what is happening. I changed the structure and some names because we got import errors during publishing. These imports point to the old structure